website/docs/docs/cloud-integrations/configure-auto-exposures.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Auto-exposures"
+id: "configure-auto-exposures"
+sidebar_label: "Configure auto-exposures"
+description: "Import and auto-generate exposures from dashboards and understand how models are used in downstream tools for a richer lineage."
+---
+
+# Configure auto-exposures <Lifecycle status='beta' />
+
+As a data team, it’s critical that you have context into the downstream use cases and users of your data products. [Auto-exposures](/docs/collaborate/auto-exposures) integrates natively with Tableau (Power BI coming soon) and [auto-generates downstream lineage](/docs/collaborate/auto-exposures#view-auto-exposures-in-dbt-explorer) in dbt Explorer for a richer experience.
+
+:::info Available in beta
+Auto-exposures is currently in beta to dbt Cloud Enterprise accounts. If you have any feedback or would like to take part in the beta, please reach out to your account manager.
+:::
+
+Auto-exposures help data teams optimize their efficiency and ensure data quality by:
+
+- Helping users understand how their models are used in downstream analytics tools to inform investments and reduce incidents — ultimately building trust and confidence in data products.
+- Importing and auto-generating exposures based on Tableau dashboards, with user-defined curation.
+- Enabling the active exposure work to run models based on when exposures are updated or need to be updated, improving timeliness and reducing costs.
+
+## Prerequisites
+
+To access the features, you should meet the following:
+
+1. You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
+2. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
+3. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings
+4. Use Tableau as your BI tool and can enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled. 
+
+## Set up in Tableau
+
+This section of the document explains the steps you need to set up the auto-exposures integration with Tableau. Once you've set this up in Tableau and dbt Cloud, you can view the [auto-exposures](/docs/collaborate/auto-exposures#view-auto-exposures-in-dbt-explorer) in dbt Explorer.
+
+To set up personal access tokens (PATs) needed for auto exposures, ask a site admin to configure it for the account. 
```

Comment:
Hiya @mirnawong1 

Another non-blocker suggestion, would it also be worthwhile linking the user to the PAT doc: https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens#using-the-new-personal-access-tokens?

Kind Regards